steps/discover-repos.rb

```diff
@@ -51,6 +51,11 @@
 raise 'Can only retrieve up to 1000 repos' if opts[:total] > max
 
 size = [opts[:page_size], opts[:total]].min
+licenses_to_filter = [
```

Comment:
@Orillio check this out: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html